libs/core-common/src/jsonSchema/ajv.ts

```diff
@@ -8,7 +8,7 @@ export class AjvValidationService implements ValidationService {
   protected ajv: Ajv.Ajv = new Ajv();
 
   constructor(private logger: Logger) {
-    this.ajv.addFormat('file-urn', /^urn:ads:platform:file-service:v[0-9]:\/files\/[a-zA-Z0-9.-]*$/);
+    this.ajv.addFormat('file-urn', /^urn:[a-zA-Z0-9.-]+(:[a-zA-Z0-9.-]+)*$/);
```

Comment:
don't change this back - someone changed it to

/^urn:ads:platform:file-service:v[0-9]:/files/[a-zA-Z0-9.-]*$/

and it's admittedly better - I don't think one you're replacing it with works